desktop/main.js

```diff
@@ -157,40 +147,34 @@ const mainWindow = (() => {
              *   2. Modify the Access-Control-Allow-Origin header of the response to match the "real" origin of our Electron app.
              */
+            const webRequest = browserWindow.webContents.session.webRequest;
             const validDestinationFilters = {urls: ['https://*.expensify.com/*']};
-            if (!ELECTRON_ENVIRONMENT.isDev()) {
-                const newDotURL = ELECTRON_ENVIRONMENT.isProd() ? 'https://new.expensify.com' : 'https://staging.new.expensify.com';
-
+            /* eslint-disable no-param-reassign */
+            if (!__DEV__) {
                 // Modify the origin and referer for requests sent to our API
-                browserWindow.webContents.session.webRequest.onBeforeSendHeaders(validDestinationFilters, (details, callback) => {
-                    // eslint-disable-next-line no-param-reassign
-                    details.requestHeaders.origin = newDotURL;
-                    // eslint-disable-next-line no-param-reassign
-                    details.requestHeaders.referer = newDotURL;
+                webRequest.onBeforeSendHeaders(validDestinationFilters, (details, callback) => {
+                    details.requestHeaders.origin = CONFIG.EXPENSIFY.URL_EXPENSIFY_CASH;
+                    details.requestHeaders.referer = CONFIG.EXPENSIFY.URL_EXPENSIFY_CASH;
                     callback({requestHeaders: details.requestHeaders});
                 });
 
                 // Modify access-control-allow-origin header for the response
-                browserWindow.webContents.session.webRequest.onHeadersReceived(validDestinationFilters, (details, callback) => {
-                    // eslint-disable-next-line no-param-reassign
+                webRequest.onHeadersReceived(validDestinationFilters, (details, callback) => {
                     details.responseHeaders['access-control-allow-origin'] = ['app://-'];
                     callback({responseHeaders: details.responseHeaders});
                 });
             }
 
-            if (ELECTRON_ENVIRONMENT.isDev()) {
-                require('dotenv').config();
-                if (process.env.USE_WEB_PROXY !== 'false') {
-                    browserWindow.webContents.session.webRequest.onHeadersReceived(validDestinationFilters, (details, callback) => {
-                        // eslint-disable-next-line no-param-reassign
-                        details.responseHeaders['access-control-allow-origin'] = ['http://localhost:8080'];
-                        callback({responseHeaders: details.responseHeaders});
-                    });
-                }
+            if (__DEV__) {
+                webRequest.onHeadersReceived(validDestinationFilters, (details, callback) => {
+                    details.responseHeaders['access-control-allow-origin'] = [`http://localhost:${port}`];
```

Comment:
Should this be like so:

```suggestion
                    details.responseHeaders['access-control-allow-origin'] = [`http://localhost:${process.env.PORT}`];
```

Reply:
on line 25 of `start.js` we set the `PORT` environment variable in all caps.

Reply:
I still see it in lowercase on `main`
https://github.com/Expensify/App/blob/cc90f5668f3be65f7b19db84ba3db654338812c4/desktop/main.js#L18